jwst/regtest/test_nirspec_fs_spec2.py

```diff
@@ -35,7 +35,7 @@
 
 
 @pytest.fixture(scope="module", params=file_roots)  # ids=ids)
-def run_pipeline(jail, rtdata_module, request):
+def run_pipeline(tmp_cwd_module, rtdata_module, request):
```

Comment:
For all these cases where the `rtdata_module` fixture is used, `tmp_cwd_module` is not needed, as its already included within `rtdata_module`.  Not a bug, but not necessary.

Same with tests that use the `rtdata` fixture - no need to use `tmp_cwd`, as it's already loaded as a fixture when `rtdata` gets loaded.  It's fine to use if you need its return value, but otherwise, it's redundant.

This is not a problem introduced by this PR, as it seems there were already a bunch of tests that used `_jail, rtdata` and `jail, rtdata_module` together already.

But it might be best to remove this pattern so that it is not copied further.  =)

Reply:
good point, yes, there are a lot of these instances.  Hopefully I'll get all of them